packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,706 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+
+With the upcoming v3 release, we will be introducing fluid spacers alongside our static spacers. These spacers will
+adjust in size based on a min and max value, providing a fluid response to changes in browser width.
+
+```diff
++ $pds-spacing-fluid-x-small
++ $pds-spacing-fluid-small
++ $pds-spacing-fluid-medium
++ $pds-spacing-fluid-large
++ $pds-spacing-fluid-x-large
++ $pds-spacing-fluid-xx-large
+```
+
+Additionally, we've switched from using _rem_ to _px_ for our spacers to ensure they don't unnecessarily expand when the
+root font size is adjusted.
+
+```diff
+- $p-layout-x-small
++ $pds-spacing-static-x-small
+
+- $p-layout-small
++ $pds-spacing-static-small
+
+- $p-layout-medium
++ $pds-spacing-static-medium
+
+- $p-layout-large
++ $pds-spacing-static-large
+
+- $p-layout-x-large
++ $pds-spacing-static-x-large
+
+- $p-layout-xx-large
++ $pds-spacing-static-xx-large
+```
+
+To get an overview ot the new spacings have a look [here](https://designsystem.porsche.com/latest/styles/spacing).
+
+#### Deprecated
+
+##### JS
+
+**Functions**
+
+- `pxToRem()` will be removed in v3 without replacement.
+- `remToPx()` will be removed in v3 without replacement.
+- `calculateLineHeight()` will be removed in v3. To set the `line-height`, use the `fontLineHeight` style provided.
+- `generateTypeScale()` will be removed in v3. Since the font sizes are now fluid, the `fontLineHeight` style and one of
+  the provided font sizes should be used.
+- `generateFontDefinition()` will be removed in v3. We provide `fontFamily`, `fontWeight`, `fontLineHeight` and multiple
+  font sizes, which should be used instead.
+
+**Colors**
+
+- all `external` colors are no longer available in v3.
+
+**Font**
+
+- `fontWeight` thin is removed as of v3.
+- the static `font.size` object is no longer provided in v3. Please use one of the predefined fluid font sizes or the
+  `fontLineHeight` style and on of the provided font sizes, e.g. `fontSizeTextSmall` to recreate it.
+
+**Spacings**
+
+- the static `spacing` object with `rem` spacings is no longer provided in v3. Please use one of the provided static or
+  fluid spacings.
+
+##### SCSS
+
+**Mixins**
+
+- `p-px-to-rem` will be removed in v3 without replacement.
+- `p-rem-to-px` will be removed in v3 without replacement.
+- `p-calculate-line-height` will be removed in v3. To set the `line-height`, use the `$pds-font-line-height` variable.
+- `p-generate-type-scale` will be removed in v3 since the font sizes are now fluid. The `$pds-font-line-height` style
+  and one of the provided font sizes should be used.
+- `p-generate-font-definition` will be removed in v3. We provide `$pds-font-family`, `$pds-font-weight`,
+  `$pds-font-line-height` and multiple font sizes, which should be used instead.
+
+**Colors**
+
+- theme `light-electric` and `dark-electric` are no longer available in v3.
+- all `external` colors are no longer available in v3.
+
+**Font**
+
+- `$p-font-weight-thin` is removed.
+
+```diff
+- $p-font-weight-thin
```

Comment:
I think diff is superfluous (not needed) here

Reply:
Sets the class onto the `<code>` tag and is needed for styling with prism.